src/scores/continuous/standard_impl.py

```diff
@@ -223,9 +224,98 @@ def correlation(
+            calculating the additive bias. All other dimensions will be reduced. As a
+            special case, 'all' will allow all dimensions to be preserved. In
+            this case, the result will be in the same shape/dimensionality
+            as the forecast, and the errors will be the error at each
+            point (i.e. single-value comparison against observed), and the
+            forecast and observed dimensions must match precisely.
+        weights: Optionally provide an array for weighted averaging (e.g. by area, by latitude,
+            by population, custom)
+
+    Returns:
+        An xarray object with the additive bias of a forecast.
+
+    """
+    error = fcst - obs
+    score = scores.functions.apply_weights(error, weights)
+    reduce_dims = scores.utils.gather_dimensions(
+        fcst.dims, obs.dims, reduce_dims=reduce_dims, preserve_dims=preserve_dims
+    )
+    score = score.mean(dim=reduce_dims)
+    return score
+
+
+def multiplicative_bias(
+    fcst: XarrayLike,
+    obs: XarrayLike,
+    *,
+    reduce_dims: XarrayLike = None,
+    preserve_dims: XarrayLike = None,
+    weights: XarrayLike = None,
+):
+    """
+    Calculates the multiplicative bias which is also sometimes called the mean error.
+
+    Most suited for forecasts that have a lower bound at 0 such as wind speed. Will return
+    a np.inf where the mean of `obs` across the dims to be reduced is 0.
+
+    Args:
+        fcst: Forecast or predicted variables.
+        obs: Observed variables.
+        reduce_dims: Optionally specify which dimensions to reduce when
+            calculating the multiplicative bias. All other dimensions will be preserved.
+        preserve_dims: Optionally specify which dimensions to preserve when
+            calculating the multiplicative bias. All other dimensions will be reduced. As a
+            special case, 'all' will allow all dimensions to be preserved. In
+            this case, the result will be in the same shape/dimensionality
+            as the forecast, and the errors will be the error at each
+            point (i.e. single-value comparison against observed), and the
+            forecast and observed dimensions must match precisely.
+        weights: Optionally provide an array for weighted averaging (e.g. by area, by latitude,
+            by population, custom)
+
+    Returns:
+        An xarray object with the multiplicative bias of a forecast.
+
+    """
+    reduce_dims = scores.utils.gather_dimensions(
+        fcst.dims, obs.dims, reduce_dims=reduce_dims, preserve_dims=preserve_dims
+    )
+    fcst = scores.functions.apply_weights(fcst, weights)
+    obs = scores.functions.apply_weights(obs, weights)
+
+    # Need to broadcast and match NaNs so that the fcst mean and obs mean are for the
+    # same points
+    fcst, obs = broadcast_and_match_nan(fcst, obs)
+    multi_bias = fcst.mean(dim=reduce_dims) / obs.mean(dim=reduce_dims)
```

Comment:
Yes, we need to do it this way to avoid many divide by zero errors blowing out the equation all the time whenever obs = 0. It's why I added the `broadcast_and_match_nan` .